processors/numberlines.go

```diff
@@ -0,0 +1,50 @@
+package processors
+
+import (
+	"fmt"
+	"strings"
+)
+
+// / Prepends consecutive number to each line of input
+type LineNumberer struct{}
+
+func (p LineNumberer) Name() string {
+	return "number-lines"
```

Comment:
I'm unsure about the name here

Reply:
So am I. Initially I wanted "line-numbers" and the matching alias "ln" would suggest creating links. And "nl" alias mimicks the "nl" binary from coreutils, so I went this path...